packages/conf-faad/conf-faad.1/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+homepage: "https://github.com/toots/faad"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+authors: "faad dev team"
+license: "GPL-2.0-or-later"
+build: ["pkg-config" "--exists" "faad2"]
+depends: [
+  "conf-pkg-config" {build}
+]
+depexts: [
+  ["faad2-dev"] {os-distribution = "alpine"}
+  ["faad2-devel"] {os-distribution = "centos" | os-distribution = "fedora" | os-family = "suse"}
+  ["faad2"] {os = "macos" & os-distribution = "homebrew" | os = "freebsd" | os-family = "arch" | os-distribution = "nixos" | os-distribution = "oraclelinux"}
```

Comment:
```suggestion
  ["faad2"] {os = "macos" & os-distribution = "homebrew" | os = "freebsd" | os-family = "arch" | os-distribution = "nixos" | os-distribution = "ol"}
```

Reply:
The names are now documented here: https://github.com/ocaml/opam-repository/wiki/Depexts-os-distribution---os-family-values

Reply:
Good to know, thanks!